ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -385,6 +385,12 @@ Verify Installed Library Version
     IF  '${status}' == 'FAIL'  Run Keyword And Continue On Failure  FAIL  "Expected ${lib} at version ${ver}, but ${value}"
     RETURN    ${status}    ${value}
 
+Verify Installed Labextension Version
+    [Arguments]  ${lib}  ${ver}
+    ${status}  ${value} =  Run Keyword And Warn On Failure  Run Cell And Check Output  !jupyter labextension list 2>&1 | grep ${lib} | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"v"); printf "%s", b[2]}'  ${ver}  #robocop: disable
```

Comment:
Something like

```
!jupyter labextension list 2>&1 | grep ${lib} | tr -s ' ' | cut -d ' ' -f 3 | tr -d 'v'
```

looks more readable for me, but that is just a matter of point of view. Keep it the way you have it now.

Reply:
I'm personally more familiar with the `awk` syntax, we can open it up for discussion with other folks and see what everyone thinks